Framework/Algorithms/src/GenerateEventsFilter.cpp

```diff
@@ -64,14 +64,14 @@ void GenerateEventsFilter::init() {
                   "(1) relative time to run start time in unit as specified property 'UnitOfTime'\n"
                   "(2) absolute time\n"
                   "Absolute time takes a string in format as 1990-01-01T00:00:00, while the relative time takes "
-                  "integer or float.");
+                  "a string representing an integer or a float number.");
```

Comment:
Can you change `float` to `floating-point value`.  The code converts that string to double:
```
    // Relative time in double.
    double inpt0 = std::stod(s_inpt0.c_str());

```